<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetMapWindow</class>
 <widget class="QWidget" name="TargetMapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>目标地图修改</string>
  </property>
  <widget class="QComboBox" name="TargetPlant_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_8">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_7">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_12">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_13">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_14">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_16">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_15">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_17">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_21">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_20">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_22">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_19">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_23">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_25">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_24">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_26">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_27">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_28">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_29">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_30">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_31">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_32">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_33">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_34">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_35">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_36">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_37">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_38">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_39">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_40">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_41">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_42">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_43">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_44">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>140</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_45">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_46">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_47">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_48">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_49">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_50">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_51">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_52">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TargetPlant_53">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="SeeingStars">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>看星星</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="ArtSunflower">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>向日葵艺术</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ArtWallnut">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>坚果艺术</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetTargetMap">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearTargetMap">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetTargetMap">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
